rfcs/xxxx-template.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: (2023-04-18)
+- RFC PR:
+
+# Summary
+Implement a category dropdown in the admin UI for both Collection & Single types.
+
+# Motivation
+The motivation behind this proposal is to simplify content organization and improve the user experience while managing content in the Strapi admin interface. Currently, users have to navigate through a long list of Collection and Single types, which can be time-consuming and cumbersome, especially when dealing with numerous content types. By introducing a category dropdown, users can easily group and filter their content types, making content management more efficient and user-friendly.
+
+# Detailed design
+The proposed solution includes the following design changes:
+
+1. Add a category dropdown in the admin UI for both Collection & Single types, enabling users to filter content types based on the selected category.
+2. Allow users to create, edit, and delete categories from the admin UI.
+2. Automatically assign newly created content types to a default category (e.g., "Uncategorized") if no category is selected.
```

Comment:
Instead of needing to create a category to default to, a content type's category could be nullable and those without a category would fall under "uncategorized". That way we don't have to worry about treating that category as a special case that can't be deleted, it can't be renamed (potentially a benefit to prevent confusion), we don't have to do a migration to initially create it and set all content-type categories to the default.

Reply:
Ye I think this would work better also for the approach I had in my mind